src/validations/profileSchema.js

```diff
@@ -0,0 +1,95 @@
+import { check } from 'express-validator/check';
+import makeUpperCase from '../helpers/makeUpperCase';
+
+const profileSchemas = {
```

Comment:
Let your validations be similar to what was done for `signup`. This looks like another app. You can improve on it but they should be the same. We aren't trying to confuse the users.

Reply:
Done.